app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationConfirmationAPI4.scala.html

```diff
@@ -5,9 +5,10 @@
     <p @pStyle>What happens next</p>
```

Comment:
Our current guidelines recommend to add a salutation like 'Dear [firstname]' to every email. I understand that this is an amendment to an existing email but it would be good if that could be added. Please see https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations